.github/actions/run-clang-format/action.yaml

```diff
@@ -0,0 +1,48 @@
+name: Run clang-format
+description: Runs clang-format and checks for any changes introduced by it
+inputs:
+  failCondition:
+    description: Should checks result in error response
```

Comment:
I'm having trouble understanding how this should be read or interpreted.

Reply:
In essence, you can choose for the check not to result in a failed workflow run.